src/icons/Bluetooth.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Svg, { Path, SvgProps } from 'react-native-svg';
+
+const Bluetooth: React.FC<SvgProps> = props => (
+  <Svg viewBox="0 0 32 32" {...props}>
```

Comment:
make sure default viewbox is after splatting props because props has a default of "0 0 24 24" on it

Reply:
Fixed in latest commit.

Reply:
Fixed in latest commit.